libkbfs/folder_block_manager_test.go

```diff
@@ -381,6 +381,8 @@ func TestQuotaReclamationDeletedBlocks(t *testing.T) {
 	}
 
 	// Stall the puts that comes as part of the sync call.
+	oldBServer := config2.BlockServer()
+	defer config2.SetBlockServer(oldBServer)
```

Comment:
Because otherwise `StateChecker` fails because it's not allowed to state check on something that's not a `BlockServerLocal`.